src/shared/components/navbar/Navbar.tsx

```diff
@@ -98,6 +98,13 @@ const Navbar = () => {
           type: 'link-list-icon',
           alignRight: true,
           children: [
+            {
+              type: 'link',
+              label: `${t('user.login.success')}${user?.givenName} ${user?.familyName}`,
```

Comment:
I think we should bold the text for the user's name

Reply:
Since whatever put in the label tag will be converted into string, the bold method won't work in this case. What should I do to make sure that only the username part is rendered as bold in the label tag? 
Could anyone give me a hint?

Reply:
You're right. There is no way to do this right now. This feedback can be ignored and we'll open some more issues to get this functionality added.